pkg/git/repository.go

```diff
@@ -91,6 +93,7 @@ func AddFlags(cmd *cobra.Command, bfs billy.Filesystem, prefix string) *CloneOpt
 
 	if prefix == "" {
```

Comment:
This if is ugly, can you calculate the `prefix` before and make it so there is no duplication of this flags?